rpcs3/Emu/Cell/Modules/cellRtc.cpp

```diff
@@ -1,4 +1,4 @@
-#include "stdafx.h"
+﻿#include "stdafx.h"
```

Comment:
I have register this in first time in February, for that, just prefer with this correct order